From 07e75dc07f31d859ce3b80e6980a3cb0ef781976 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sat, 19 Jul 2014 04:27:27 -0400
Subject: [pci] Update documentation 00-INDEX file

Message-id: <20140719042726.19438.42485.stgit@gir.stowe>
Patchwork-id: 86007
O-Subject: [RHEL7.1 PATCH 45/81] PCI: Update documentation 00-INDEX file
Bugzilla: 1110895
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110895
Upstream Status: c2a5a829e28c476880b5efc8755ab3b339fdc147

commit c2a5a829e28c476880b5efc8755ab3b339fdc147
Author: Erik Ekman <erik@kryo.se>
Date:   Wed Jan 1 22:15:25 2014 +0100

    PCI: Update documentation 00-INDEX file

    The PCI-DMA-mapping.txt moved to general docs and became DMA-API-HOWTO.txt
    in 5e07c2c7301b ("Documentation: rename PCI/PCI-DMA-mapping.txt to
    DMA-API-HOWTO.txt").  Add new file about PCI Express I/O Virtualization.

    Signed-off-by: Erik Ekman <erik@kryo.se>
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/Documentation/PCI/00-INDEX b/Documentation/PCI/00-INDEX
index 812b17f..147231f 100644
--- a/Documentation/PCI/00-INDEX
+++ b/Documentation/PCI/00-INDEX
@@ -2,12 +2,12 @@
  - this file
 MSI-HOWTO.txt
  - the Message Signaled Interrupts (MSI) Driver Guide HOWTO and FAQ.
-PCI-DMA-mapping.txt
- - info for PCI drivers using DMA portably across all platforms
 PCIEBUS-HOWTO.txt
  - a guide describing the PCI Express Port Bus driver
 pci-error-recovery.txt
  - info on PCI error recovery
+pci-iov-howto.txt
+ - the PCI Express I/O Virtualization HOWTO
 pci.txt
  - info on the PCI subsystem for device driver authors
 pcieaer-howto.txt
-- 
1.7.1